thorlcr/master/thmastermain.cpp

```diff
@@ -967,6 +967,23 @@ int main( int argc, const char *argv[]  )
                 throwStringExceptionV(0, "Failed to connect to all nodes");
             PROGLOG("verified mp connection to rest of cluster");
 
+#ifdef _CONTAINERIZED
+            if (globals->getPropBool("@_dafsStorage"))
+            {
+                std::vector<std::string> hosts;
+                for (unsigned n=0; n<queryProcessGroup().ordinality(); n++)
+                {
+                    INode &node = queryProcessGroup().queryNode(n);
+                    StringBuffer host;
+                    node.endpoint().getIpText(host);
+                    hosts.push_back(host.str());
+                }
+                queryNamedGroupStore().add(thorName, hosts);
+                // change default plane
+                queryComponentConfig().setProp("storagePlane", thorName);
```

Comment:
This should also be creating a storageplane entry - otherwise it will cause problems in the next stages of the disk refactoring.

Reply:
As discussed, it can't create a real SP from a running instance dynamically.
This will either require re-thinking once DFS is refactored.